libs/blocks/global-navigation/features/profile/dropdown.js

```diff
@@ -162,6 +165,8 @@ class ProfileDropdown {
 
     signOutLink.addEventListener('click', (e) => {
       e.preventDefault();
+      document.cookie = `${ACOM_SIGNED_IN_STATUS}=;path=/;expires=${new Date(0).toUTCString()};`;
```

Comment:
I don't think this approach is correct, it will only work on MWP pages, but the users can sign out from different sources like behance.net, creativecloud.adobe.com, photoshop web, express, etc. Those surfaces don't use Milo or FEDS on AEM, they use the standalone Profile Menu or a different implementation which will not remove the cookie. Use-cases like the one below will break:
* go to adobe.com and sign in -> cookie is set
* go to a page which is not Milo / FEDS on AEM and sign out -> cookie is not removed
* go to adobe.com -> cookie is still set, issue

Additionally, we will roll out the Universal Navigation which will replace the current code you have changed; a new Profile Menu will be loaded from an external library and we will not be able to set the custom cookie logic.